src/index.js

```diff
@@ -34,11 +35,14 @@ export function call(obj, args, cons = console){
     }
 }
 
-export function run(cmd, options){
-    options = options || {};
+export function run(cmd, options = {}){
```

Comment:
I don't think so. If options are provided then the default won't take affect and env wouldn't be defined.